### PR TITLE
make our test suite as much warnings free as possible

### PR DESCRIPTION
**Pull request by gdementen**

### CODE REVIEW

File: larray/core/array.py [hunk 1]
```diff
@@ -3601,7 +3603,7 @@ def rationot0(self, *axes):
 
         for reference, the normal ratio method would produce a warning message and return:
 
-        >>> arr.ratio('a')
+        >>> arr.ratio('a')                                          # doctest: +SKIP
```
Comment (alixdamman) [inline larray/core/array.py:3606]: Is it tested in our unittests ?

File: larray/core/array.py [hunk 2]
```diff
@@ -5866,14 +5870,17 @@ def divnot0(self, other):
...
-             BE  nan  1.0
-             FO  inf  3.0
         >>> a.divnot0(b)
         nat\sex    M    F
              BE  0.0  1.0
              FO  0.0  3.0
+
+        Compare this to:
+
+        >>> a / b                                  # doctest: +SKIP
```
Comment (alixdamman) [inline larray/core/array.py:5880]: Is it tested in our unittests ?

**Review by alixdamman — APPROVED** [review]

**Comment by alixdamman** [comment]
Hum... Travis tests raise the following error:
> ModuleNotFoundError: No module named 'pywintypes'

File: larray/core/array.py (hunk 1 above)
Comment (gdementen) [reply larray/core/array.py:3606]: Not sure about unit tests but it is at least tested in the doctest of the ratio method (this is rationot0).

File: larray/core/array.py (hunk 2 above)
Comment (gdementen) [reply larray/core/array.py:5880]: > Hum... Travis tests raise the following error:
> 
> > ModuleNotFoundError: No module named 'pywintypes'

I know. I introduced Windows specific code without noticing. I'll fix this next week.

**Comment by gdementen** [comment]
> Hum... Travis tests raise the following error:
> 
> > ModuleNotFoundError: No module named 'pywintypes'

I know. I introduced Windows specific code without noticing. I'll fix this next week.

File: larray/core/array.py (hunk 2 above)
Comment (gdementen) [reply larray/core/array.py:5880]: No idea... I will check that next week.

File: larray/core/array.py (hunk 1 above)
Comment (alixdamman) [reply larray/core/array.py:3606]: OK

**Comment by gdementen** [comment]
hmmmfff... Travis doesn't seem to pickup my changes...

**Comment by gdementen** [comment]
oh, it was picked up after all and I triggered a double build 😲.

File: larray/core/array.py (hunk 2 above)
Comment (gdementen) [reply larray/core/array.py:5880]: yes it is (in test_array.py/test_binary_ops)

**Pull request merged**